assets/FontDownloads/License.txt

```diff
@@ -0,0 +1,37 @@
+
+
+�X------------------------------------------------------�[
+��                                                    ��
+��     FREE FONT : http://www.OnlineWebFonts.Com      ��

```

Comment:
Download the font straight from Google: https://fonts.google.com/specimen/Manrope > Download Family